docs/usermanual.md

```diff
@@ -843,6 +842,38 @@ the example case the command will be:
 }
 ```
 
+The agent provides additional information, hereafter referred as **JEXL context**, in order to be used to evaluate the
+expression of the **command** as well as the **device endpoint** when apply. In all cases the following data is available to all expressions within the JEXL context:
+
+-   `id`: device ID
+-   `type`: NGSI entity type (principal)
+-   `entity_name`: Entity Name
+-   `service`: device service (`Fiware-Service`)
+-   `subservice`: device subservice (`Fiware-ServicePath`)
+-   `staticAttributes`: static attributes defined in the device or config group (flatted into context object).
+-   `attribute`: attribute name and value involved in the command.
+
+An example of JEXL context could be like:
+
+```json
+{"id":"disp2","type":"thing","service":"smartcity","subservice":"/","entity_name":"thing:disp2","color":"blue","brand":"tef","reset":0}
+```
+Where *color* and *brand* are static attrs and *reset* is the command name and *0* is the value for the command.
```

Comment:
I understand it is not a way of distinguish static attributes (color, brand) from the command involved in the command (reset). Is my understanding correct?